### DISCUSSION TITLE
CORS issues

### DISCUSSION THREAD

**Discussion by helghast79** [tauri-apps/tauri#1100] [General]
Some javascript libraries use XMLHttpRequest to exchange data with the server and they fail due to CORS.
Tauri http works but how can it override XMLHttpRequest so that these libraries work without change?

**Comment by MrOggy85** [comment]
I think it uses [curl](https://crates.io/crates/curl) under the hood. Basically the request is not made in the webview. It is done basically in the same way as if you would use curl fro, the terminal

**Comment by samirdjelal** [comment]
You can set `mode: 'no-cors'`

```js
fetch("https://localhost", {
  method: "POST",
  responseType: 1,
  mode: 'no-cors',
  body: Body.json({id: 1}),
  headers: {"Accept": "application/json", "Content-Type": "application/json"}
})
